### DISCUSSION TITLE
How to set specific routes to load local HTML files in Nuxt.js without recompiling?

### DISCUSSION THREAD

**Discussion by wscart** [nuxt/nuxt#24883] [Questions]
### Describe the feature

How to avoid route conflicts in Nuxt.js projects that combine SPA and SSR?

How to combine existing SPA and SSR projects?

In a project that combines SPA and SSR, there may be route conflicts when accessing certain routes. To resolve this issue, one solution is to set a specific path in Nuxt.js router for loading an HTML file while avoiding conflicts with other paths. For example, the path "htttp://xxx.com/app" can be used to load the HTML file, while all other paths will be rendered through the Nuxt.js router.

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [ ] Read the [contribution guide](https://nuxt.com/docs/community/contribution).
- [ ] Check existing [discussions](https://github.com/nuxt/nuxt/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by TheAlexLichter** [comment]
The solution needs two tiers:

"Server-side": Ensure that you proxy / load URLs for both apps correctly (e.g. let your webserver handle the paths differently).
"Client-side": Use external links only to switch from App 1 to App 2, so the server-side proxy will work correctly.
